src/Jwt.ts

```diff
@@ -0,0 +1,65 @@
+import * as atlassianJwt from 'atlassian-jwt';
```

Comment:
I got rid of the `util.ts` file and moved the code to more specific places. For instance:

- This file contains helper methods to decode JWTs
- The `requestHeader` directory contains helpers to parse information from the request object